pkg/skaffold/schema/latest/config.go

```diff
@@ -148,6 +148,7 @@ type DeployType struct {
 	HelmDeploy      *HelmDeploy      `yaml:"helm,omitempty" yamltags:"oneOf=deploy"`
 	KubectlDeploy   *KubectlDeploy   `yaml:"kubectl,omitempty" yamltags:"oneOf=deploy"`
 	KustomizeDeploy *KustomizeDeploy `yaml:"kustomize,omitempty" yamltags:"oneOf=deploy"`
+	MortarDeploy    *MortarDeploy    `yaml:"mortar,omitempty" yamltags:"oneOf=deploy"`
```

Comment:
Adding new fields to the yaml now requires creating a new version of the schema. I can help you with that once everything else is ok.